pom.xml

```diff
@@ -655,15 +649,16 @@ under the License.
               <goal>enforce</goal>
             </goals>
             <phase>validate</phase>
```

Comment:
Is this check still required with this update?

Reply:
I have no idea about proliferation of this dependency, so it MAY be needed. HOPEFULLY is not needed 😄 I think is okay to leave it after we ensure nothing pulls in these.